Documentation/teaching/info/vm.rst

```diff
@@ -150,12 +150,3 @@ the kernel by running the *gdb* target from tools/labs:
    #9  0xc10001da in startup_32_smp () at arch/x86/kernel/head_32.S:368
    #10 0x00000000 in ?? ()
    (gdb)
-
-Rebuild the kernel image
```

Comment:
As I mentioned previously, removing the bzImage file is required if we want to rebuild the kernel image.

Reply:
OK. I'll test and update it.